### PR TITLE
Change spelling of CDT Cloud

### PR DESCRIPTION
**Pull request by planger**
Now as the rename of the project has been approved (see https://gitlab.eclipse.org/eclipsefdn/emo-team/emo/-/issues/440), we can apply the changes to the website.

### CODE REVIEW

File: content/_index/blueprint/index.md [hunk 1]
```diff
@@ -14,16 +14,16 @@ background = "dark"
...
 
 <div style='background: lavender;color: black;padding: 1rem'>
 
-CDT.cloud Blueprint is in an early alpha state and undergoing active development
+CDT Cloud Blueprint is in an early alpha state and undergoing active development
 
 * Check out the [development plan](https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint/milestone/1) for Release 1.0
-* For the current up to date development version of CDT.cloud Blueprint including a Docker based development build, see the [Github repository](https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint)
+* For the current up to date development version of CDT Cloud Blueprint including a Docker based development build, see the [Github repository](https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint)
 * Prereleases of CDT.Cloud Blueprint are available to be [downloaded and installed](https://download.eclipse.org/theia/cdt-cloud/)
```
Comment (sdirix) [inline content/_index/blueprint/index.md:27]:
```suggestion
* Prereleases of CDT Cloud Blueprint are available to be [downloaded and installed](https://download.eclipse.org/theia/cdt-cloud/)
```

File: content/documentation/additionals/content.md [hunk 2]
```diff
@@ -8,18 +8,18 @@ title = "External Articles & Videos"
...
+* <a target="_blank" href="https://eclipsesource.com/blogs/2022/01/10/cdt-cloud-an-overview-about-c-c-tooling-in-the-web/">CDT Cloud – An overview about C/C++ tooling in the web</a>
 
-Also, we provide a list of talk recordings and videos on CDT.cloud:
+Also, we provide a list of talk recordings and videos on CDT Cloud:
 
-* EclipseCon 2022: <a target="_blank" href="https://www.youtube.com/watch?v=zDDAu2LDxSs">Introducing Eclipse CDT.cloud - C/C++ tooling in the web</a>
+* EclipseCon 2022: <a target="_blank" href="https://www.youtube.com/watch?v=zDDAu2LDxSs">Introducing Eclipse CDT Cloud - C/C++ tooling in the web</a>
 * EclipseCon 2022: <a target="_blank" href="https://www.youtube.com/watch?v=sNdsuxE5Q1Y">What's cooking in the TraceCompass cloud project</a>
 * Cloud IDE Days 2022: <a target="_blank" href="https://www.youtube.com/watch?v=zRZrv4c6lhk">Building Custom C/C++ Tools in the Web</a>
 * CDT.cloud Overview: <a target="_blank" href="https://www.youtube.com/watch?v=H3E46x8MzzQ">Elevator pitch</a>
```
Comment (sdirix) [inline content/documentation/additionals/content.md:21]:
```suggestion
* CDT cloud Overview: <a target="_blank" href="https://www.youtube.com/watch?v=H3E46x8MzzQ">Elevator pitch</a>
```

File: content/documentation/additionals/content.md [hunk 3]
```diff
@@ -8,18 +8,18 @@ title = "External Articles & Videos"
...
 * CDT.cloud Overview: <a target="_blank" href="https://www.youtube.com/watch?v=1aO0pxCcUrk">Walkthrough</a>
```
Comment (sdirix) [inline content/documentation/additionals/content.md:22]:
```suggestion
* CDT cloud Overview: <a target="_blank" href="https://www.youtube.com/watch?v=1aO0pxCcUrk">Walkthrough</a>
```

File: content/documentation/additionals/content.md [hunk 4]
```diff
@@ -8,18 +8,18 @@ title = "External Articles & Videos"
...
 * Trace Compass.cloud: <a target="_blank" href="https://www.youtube.com/playlist?list=PL9c8Jxzvk1-lTdH79COvnzCwEPt3BFfo_">Several videos on Trace Compass.cloud 8.1</a>
```
Comment (sdirix) [inline content/documentation/additionals/content.md:23]: Doesn't Trace Compass also change their name?
```suggestion
* Trace Compass Cloud: <a target="_blank" href="https://www.youtube.com/playlist?list=PL9c8Jxzvk1-lTdH79COvnzCwEPt3BFfo_">Several videos on Trace Compass Cloud 8.1</a>
```

**Review by sdirix — COMMENTED** [review]
Looks good! However I found some more instances.

Also in `mail.md` we still offer a `cdt.cloud@eclipsesource.com` mail address. Can/Did we change this?

**Comment by planger** [comment]
Great, thank you very much!

**Review by sdirix — APPROVED** [review]
No change for the email address?

**Comment by planger** [comment]
> cdt.cloud@eclipsesource.com

I have a task to change the email address and will then update the web page. Thanks!

**Pull request merged**